src/styles/headings/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Headings
+description: Use headings to structure content on pages
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+showPageNav: true
+order: 7
+---
+
+{% from "_example.njk" import example %}
+
+Write all headings in sentence case.
+
+Use heading tags, such as `<h1>`, `<h2>` and so on, to tag the headings on a page. Apply a heading class, such as `govuk-heading-l`, to style them visually. Style headings consistently to create a clear content structure throughout your service.
+
+For a [question page](/patterns/question-pages/), or pages with long headings, start with `govuk-heading-l` for an `<h1>`, `govuk-heading-m` for an `<h2>` and so on. But change it if your pages feel unbalanced – the heading class you use does not always need to correspond to the heading level.
```

Comment:
We talk about heading tags and heading class, and then this para also has heading level. Is the heading level a different thing or related to a heading tag? And do we need to define what a heading level is?